README.md

```diff
@@ -0,0 +1,93 @@
+# FlatMate - iOS Roommate Matching App
+
+FlatMate is a modern iOS application designed to help users find compatible roommates through a streamlined matching process.
+
+## Features
+
+### Smart Matching System
+- Profile-based matching algorithm
+- Preferences for smoking, pets, noise tolerance, and more
+- Location-based roommate search
+
+### User Authentication
+- Secure email/password registration and login
+- Profile creation and management
+- Password reset functionality
+
+### Interactive UI
+- Swipe-based matching interface
+- Real-time chat functionality
+- Modern, intuitive design
+- Comprehensive onboarding process
+
+### Profile Management
+- Customizable user profiles
+- Profile picture upload
+- Lifestyle preferences settings
+- Location preferences
+
+### Messaging System
+- Real-time chat with matches
+- Message notifications
+- Chat history management
+
+## Technical Stack
+
+### Frontend
+- SwiftUI for UI components
+- iOS 18.0+ support
+
+### Backend
+- Firebase Authentication
+- Firebase Firestore for database
+- Firebase Storage for media
+
+## Requirements
+- iOS 18.0 or later
+- Xcode 16.0 or later
+- Firebase account for backend services
+
+## Installation
+
+1. Clone the repository
+```bash
+git clone https://github.com/yourusername/FlatMate.git
+```
+
+2. Install dependencies
+```bash
+cd FlatMate
```

Comment:
```suggestion
cd FlatMates
```

Reply:
The folder name is actually called FlatMate lol